--- a/third_party/libaom/BUILD.gn
+++ b/third_party/libaom/BUILD.gn
@@ -32,10 +32,9 @@ source_set("libaom") {
 }
 
 config("libaom_rc_link_settings") {
   include_dirs = [ "/work/external/libaom-rtc" ]
-  ldflags = [
-    rebase_path("/work/external/libaom-rtc/out-rtc/libaom_av1_rc.a", root_build_dir),
-    rebase_path("/work/external/libaom-rtc/out-rtc/libaom.a", root_build_dir),
+  libs = [
+    "/work/external/libaom-rtc/out-rtc/libaom_av1_rc.a",
+    "/work/external/libaom-rtc/out-rtc/libaom.a",
   ]
 }
 
