--- a/build/linux/unbundle/libvpx.gn
+++ b/build/linux/unbundle/libvpx.gn
@@ -1,9 +1,13 @@
 pkg_config("system_libvpx") {
   packages = [ "vpx" ]
 }

+config("libvpx_internal_headers") {
+  include_dirs = [ "//third_party/libvpx/source/libvpx" ]
+}
+
 shim_headers("libvpx_shim") {
   root_path = "source/libvpx"
   headers = [
     "vpx/vp8.h",
     "vpx/vp8cx.h",
@@ -11,11 +15,12 @@
     "vpx/vpx_encoder.h",
     "vpx/vpx_frame_buffer.h",
     "vpx/vpx_image.h",
     "vpx/vpx_integer.h",
+    "vpx/internal/vpx_ratectrl_rtc.h",
   ]
 }

 source_set("libvpx") {
   deps = [ ":libvpx_shim" ]
-  public_configs = [ ":system_libvpx" ]
+  public_configs = [ ":system_libvpx", ":libvpx_internal_headers" ]
 }
